components/Dialogs/src/Theming/UIViewController+MaterialTheming.h

```diff
@@ -16,18 +16,17 @@
 #import "MaterialContainerScheme.h"
 
 /**
- This category is used to style MDCDialogPresentationController instances to a specific Material
- style which can be found within the [Material
- Guidelines](https://material.io/design/components/dialogs.html).
+ This category adds methods to theme the presentation style of custom UIViewControllers
+ when they are presented by MDCDialogPresentationController. The presentation theming follows
+ the Material spec set in [Material Guidelines](https://material.io/design/components/dialogs.html).
  */
-@interface MDCDialogPresentationController (MaterialTheming)
+@interface UIViewController (MaterialTheming)
 
 /**
- Applies a container scheme to this instance.
+ Applies a container scheme to a custom view controller, to theme its presentation style
 
- @param scheme A container scheme instance containing any desired customizations to the theming
- system.
+ @param scheme A container scheme used for customizing the theme
  */
-- (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme;
+- (void)applyPresentationThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme;
```

Comment:
methods added to UIKit classes via categories should have a prefix.
mdc_applyPresentationThemeWithScheme: